crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -59,6 +63,25 @@ impl AlgorithmV1 {
     }
 }
 
+pub type Height = u32;
```

Comment:
```suggestion
pub struct Height(u32);
```

nit: I usually try do advocate for the `Newtype` pattern (see https://www.lurklurk.org/effective-rust/newtype.html).
In this case I think that we have this type defined somewhere else already https://github.com/FuelLabs/fuel-core/blob/773226cb326c2607d274717348ff50aab30b0828/crates/types/src/blockchain/primitives.rs#L111

Reply:
Yeah. I don't want to use foreign types here. I want to keep the domain independent.

I'm not completely sold on using newtypes for situations like this. I mostly just added this for readability on the `BTreeMap` generics, otherwise people would just be guessing what the `u32` and `u64` represent. But the newtype would add an additional construct or `From` conversion for the caller, an that's noisy IMO.